frontend/src/app/signUp/IdentityVerification/PersonalDetailsForm.tsx

```diff
@@ -78,8 +81,23 @@ const PersonalDetailsForm = () => {
     setSaving(false);
   };
 
+  if (orgExternalId === null) {
+    return (
+      <CardBackground>
+        <Card logo bodyKicker={"Invalid request"} bodyKickerCentered={true}>
+          <p className="text-center">No organization ID found in the URL.</p>
```

Comment:
If this is going to be shown to end users, maybe something more generic like "We weren't able to find your affiliated organization"?

Reply:
Updated in 460b45d